package.json

```diff
@@ -10,7 +10,7 @@
     "async": "1.5.2",
     "backoff": "2.5.0",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#ce906fb5ac40b93959c02f7008c3ea7de809dfd6",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#a0936a54a42fc47403a1a3d24d2b3fb7b762b620",
```

Comment:
Now that you've pushed sdc-designation this is a stable commit hash, right?

Reply:
Yes, that's why this one was waiting on sdc-designation. After it was merged I grabbed the hash and added it here. Previously it was just `.../sdc-designation.git#linuxcn`, but now that we're heading back toward master I wanted it to be a specific commit hash again.